index.html

```diff
@@ -3,11 +3,16 @@
 
 <head>
     <title>Project UFEM</title>
+    <link rel="stylesheet" type="text/css" href="./css/styles.css" />
```

Comment:
_Void elements_ (_HTML_ elements that cannot have any content) must not have specified end tag.
Also following the [_MDN_](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link#attr-type) regarding `type` attribute:

> The common use of this attribute is to define the type of stylesheet being referenced (such as text/css), but given that CSS is the only stylesheet language used on the web, not only is it possible to omit the type attribute, but is actually now recommended practice.

I think we might omit it at the moment(?).

```suggestion
    <link rel="stylesheet" href="./css/styles.css">
```

You can read more about _void element_ at:
- https://www.w3.org/TR/2011/WD-html-markup-20110113/syntax.html#void-element